rxandroid/src/main/java/rx/android/app/AppObservable.java

```diff
@@ -62,10 +61,10 @@ public Boolean call(android.support.v4.app.Fragment fragment) {
 
     /**
      * Binds the given source sequence to an activity.
-     * <p>
+     * <p/>
```

Comment:
This is wrong, revert. Paragraphs are not self-closing and doclint on JDK 8 will fail the build for this.